src/Thorn_Calculus-Core_Bisimilarities.thy

```diff
@@ -1350,9 +1350,148 @@ text \<open>
   commutativity rules.
 *)
 
+lemma repeated_receive_evolution:
```

Comment:
I think, we should name this lemma `transition_from_repeated_receive`, in correspondence with similar lemmas, like `sending_transition_from_adapted`.

Reply:
Also I think that, because this lemma is rather fundamental and potentially usable in other places, we should move it into `Thorn_Calculus-Semantics-Synchronous`, preferably right before `adapted_io_transition`.

Reply:
Fixed in f1d1fc4712732b2ef29314be5e7c0846b639c6e8 and f62d0237aee8a130b575774a1fa9c23e19b1b2d8, please verify.

Reply:
Looks good. Thank you.